### DISCUSSION TITLE
Aptx and Aptx HD not working on Pixel 6

### DISCUSSION THREAD

**Discussion by jhoynes01** [AndroidAudioMods/ViPERFX_RE#14] [General]
Pixel 6 Aptx and Aptx HD No processing on Bluetooth. changing the codec to SBC and AAC works around this issue but at a cost of audio quality. Thanks to the developer for working on this project and giving us the ability to have viper once again many thanks!

**Comment by iscle** [comment]
Hi,

Please post an screenshot of the driver status dialog when playing over aptX please.
